doc/README

```diff
@@ -13,23 +19,49 @@ optionally, you might need for special features:
  - librsvg2 (for watermark plugin)
  - flickcurl (for Flickr support)
  - libGL/sdl12 (to build slideshow viewer)
- - dbus-glib, gnome-keyring (kwallet/gnome keyring support)
- - osm-gps-map for geo tagging view
+ - dbus-glib
+ - osm-gps-map and libsoup for geo tagging view
  - lensfun (lens distortion plugin)
  - OpenEXR for HDR export
-
-other used packages (supplied in the source tree):
-
- - LibRaw nikon_curve (taken from ufraw)
- - RawSpeed
- - squish
+ - SDL, SDL-image and Mesa OpenGL for the slideshow view
+ - flickcurl for uploading direct to Flickr
+ - libsecret for storing passwords
+ - libcolord-dev libcolord-gtk-dev for colour profile support
+ - webp and openjpeg libraries for WebP and JPEG 2000 support
+ - GraphicsMagick library for TIFF EXIF and LDR image format support
 
 then, type:
 $ ./build.sh --prefix /usr --buildtype Release
 $ cd build && make install (or sudo make install)
 $ darktable
 
-and enjoy!
+optionally, to build the user manual:
+
+ - Java JDK, gnome-doc-utils, Saxon 6.5.x, FOP and ImageMagick
+ - xsltproc and the DocBook XML DTD and XSL stylesheets
+
+then, type:
+$ cd build
+$ make darktable-usermanual
+$ evince doc/usermanual/darktable-usermanual.pdf
+
+optionally, to build translations of the manual pages:
+
+ - PO for anything (po4a)
+
+other used packages (supplied in the source tree):
+
+ - RawSpeed
+ - Lua 5.2 and LuaAutoc
```

Comment:
I found while I was testing the build on multiple platforms that I used the included Lua 5.2 more often than not, and I didn't have to tell cmake to do anything differently (I generally build with build.sh).